migrations/20230501-init-schema.sql

```diff
@@ -0,0 +1,49 @@
+CREATE DATABASE quiz_webapp;
```

Comment:
[IMO] In order to reflesh the sql just in case of bugs or something, I think it is better to have DROP method.
```
DROP TABLE subjects;
DROP TABLE quizzes;
DROP TABLE links;
DROP TABLE questions;
```